### DISCUSSION TITLE
Listing/counting subject IDs in ABCD

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#27] [Where in the World is…? (Help)]
Does anyone have R code for listing/counting subject IDs from ABCD that show how many and which time points they have data from? I feel like this is a really basic question but also one that I could spend a long time devising something from scratch, so if there's a standard way to do it I would love to know :)

**Comment by carly-gray-19** [comment]
This is a very belated answer but I was browsing the forum and figured I'd share one potential option from the [naniar package](https://naniar.njtierney.com/) for missing data. This is sort of the inverse of what you're asking for but it's what I've found helpful in trying to figure out what was measured when and how many participants completed a measure:
```
library(dplyr)
library(naniar)
abcd5.1 %>% group_by(eventname) %>% miss_var_summary()
```
This will output a table of variables, organized by eventname, with the number and percent of rows in which that variable is missing. Other functions in the `naniar` package might also be useful to you, like visual summaries of missing data if you have a large data frame.
